apps/e2e/action-bars-storybook-e2e/src/e2e/summary-action-bar.component.cy.ts

```diff
@@ -12,119 +12,126 @@ Cypress.on(
 describe('action-bars-storybook - summary action bar', () => {
   E2eVariations.forEachTheme((theme) => {
     describe(`in ${theme} theme`, () => {
-      ['tab', 'page', 'split-view', 'modal', 'modal-full-page'].forEach(
-        (style) => {
-          const viewportSizes = [
-            ...E2eVariations.DISPLAY_WIDTHS.map((width) => [width, 960]),
-            ...E2eVariations.MOBILE_WIDTHS.map((width) => [width, 590]),
-          ];
-          // TODO: make new array with Lg and Xs widths
-          viewportSizes.forEach(([width, height]) => {
-            describe(`at ${width}px (${style})`, () => {
-              it(`should render the component at width ${width} (${style})`, () => {
+      [
+        'tab',
+        'page',
+        'split-view',
+        'modal',
+        'modal-full-page',
+        'single-error',
+        'overflow-errors',
+        'modal-errors',
```

Comment:
all i did in this file was add the last two `style` options, prettier makes it look like a lot more changed lol